src/profile_manager/tests/test_views.py

```diff
@@ -396,6 +401,34 @@ def test_profile_list_staff(self):
         qs = response.context['object_list']
         self.assertEqual(qs.count(), 3)  # self.test_teacher, deck_owner, admin
 
+    def test_profile_list_block__get_queryset(self):
+        """ProfileListBlock view's get queryset method should return a queryset containing only students in the desired block"""
```

Comment:
DOcstring doesn't mention testing inactive students, but it appears below you are checking that too?

Reply:
Test Docstring updated to better reflect inactive testing